state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -406,6 +409,7 @@ pub mod pallet {
 			asset: TargetChainAsset<T, I>,
 			amount: AssetAmount,
```

Comment:
will this still be the amount where fees have already been deducted?

Reply:
I haven't changed the behaviour here. Though from what I can tell (also having tested this on localnet now), the `amount` emitter here *includes* the egress fee.

Reply:
yes you are right, sorry. its the amount before the fee is deducted 👍